scripts/ci-entrypoint.sh

```diff
@@ -152,6 +154,16 @@ create_cluster
 # export the target cluster KUBECONFIG if not already set
 export KUBECONFIG="${KUBECONFIG:-${PWD}/kubeconfig}"
 
+# install cloud-provider-azure components, if using out-of-tree
+if [[ -n "${TEST_CCM:-}" ]]; then
+  "${HELM}" install --repo https://raw.githubusercontent.com/kubernetes-sigs/cloud-provider-azure/master/helm/repo cloud-provider-azure --generate-name --set infra.clusterName="${CLUSTER_NAME}" --set cloudControllerManager.imageRepository="${IMAGE_REGISTRY}" \
```

Comment:
@lzhecheng @jackfrancis is there still a plan to use gh pages to make an official helm repo for cloud-provider-azure?

Reply:
I'm not sure a plan was ever formulated. I'd be in favor of doing that, if the cloud-provider-azure maintainers agree.

Filed a proposal issue here:

https://github.com/kubernetes-sigs/cloud-provider-azure/issues/1418

Reply:
Good idea!